Documentation second draft (#6)
SABS-R3-Epidemiology/rEpiabm: README.md

* *Places* - quantity per microcell is based on a probabilistic distribution. These are spaces where individuals might meet other individuals from different households, a workplace or a public park for example.

> [!IMPORTANT]  
> Follow **2.1 Using EpiGeoPop** instructions for extracting a population using Epigeopop or **2.2 Using User-defined values** for a toy population.

**2.1 Using EpiGeoPop**

The structure is created using [EpiGeoPop](https://github.com/SABS-R3-Epidemiology/EpiGeoPop). The user states a region of interest, Oxford or UK for example, and the tool extracts information from [Natural Earth](https://www.naturalearthdata.com/) and [JRC](https://data.jrc.ec.europa.eu/csv), providing a csv file as output. This file contains one line per microcell for each cell, with the number of households, places and individuals to be used in the simulation (the quantity of individuals are extracted from Census data).

**Instructions:**

1. Go to [EpiGeoPop](https://github.com/SABS-R3-Epidemiology/EpiGeoPop) repository and follow the instructions to extract a csv file of your required region.

> [!NOTE]  

> > The json file which you amend to put the name your country also needs the proportion of households with 1 individual, 2 individuals, 3 individuals... upto 10 individuals. This information is usually found using census data (or equivalent) for your region. Amend the json file as described [in this Jupyter notebook](./walk_through/detailed_example.ipynb). 

Reviewer: ```suggestion
> The .json file which you amend to put the name your country also needs the proportion of households with 1 individual, 2 individuals, 3 individuals... up to 10 individuals. This information is usually found using census data (or equivalent) for your region. Amend the .json file as described [in this Jupyter notebook](./walk_through/detailed_example.ipynb). 
```

**Instructions:**

1. Open simulation.R and amend the following parameters (these are the default values):
  ```
  population_size = as.integer(100),
  cell_number = as.integer(2),
  microcell_number = as.integer(2),
  household_number = as.integer(5),
  place_number = as.integer(2)
  population_seed = as.integer(42)
  ```
2. Save simulation.R

> [!NOTE]  
> Please see more detailed instructions [in this Jupyter notebook](./walk_through/toy_example.ipynb).

### Step 3: Configure the simulation

Once the data for your region has either been extracted or configured using user-defined values, the simulation can be configured. Again, follow **3.1 Using Epigeopop population** to configure other simulation paramters for your region OR follow **3.2 Using a toy population**.

**3.1 Using Epigeopop population**

The following parameters are essential and need to be stated by the user to run a simulation:

* Name of the path to the csv file from EpiGeoPop
* Number of infected individuals (I<sub>mild</sub>: see Figure 2.): enter the number of infected individuals at the start of the simulation.

> * Proportion of households with 1 individual, 2 individuals, 3 individuals... upto 10 individuals. This information is usually found using census data (or equivalent) for your country.

Reviewer: ```suggestion
* Proportion of households with 1 individual, 2 individuals, 3 individuals... up to 10 individuals. This information is usually found using census data (or equivalent) for your country.
```

* Duration of the simulation (in days)
* Select any output options required

**Instructions:**
1. Open `Andorra_parameters.json` (copied from `Andorra` in Step 1 above) and save with `<your_country>'s` name (keep first letter capitalised).

> 2. Amend the parameter array household_size_distribution to have your countries' distribution 

Reviewer: ```suggestion
2. Amend the parameter array `household_size_distribution` to have your countries' distribution of households 
```
